docs/dl-sandbox.md

```diff
@@ -4,48 +4,49 @@ This document explains how to get started with Scalar DL Sandbox.
 
 ## Purpose of Sandbox
 
-Sandbox is for playing around with Scalar DL to roughly understand what Scalar DL does and how to write contracts.
-This is not for verifying or benchmarking its reliability, scalability and/or performance.
+The Sandbox environment is for playing around with Scalar DL to roughly understand what Scalar DL does and how to write contracts.
+This is not for verifying or benchmarking the reliability, scalability and/or performance of Scalar DL.
 If you want to interact with Scalar DL more deeply, please [contact us](https://scalar-labs.com/contact_us/).
 
 ## Get an auth token and a key pair (a certificate and a private key)
 
-From here, it assumes that you have a [GitHub](https://github.com/) account.
-If you don't, please create your account.
+From here, you will need a [GitHub](https://github.com/) account.
+If you don't have one, please create a free account.
 
-We will authorize you through GitHub OAuth to give you an access to the Sandbox environment.
-Please visit [our sandbox site](https://scalar-labs.com/sandbox/), read the [terms of use](https://scalar-labs.com/terms-of-use), and press the button to do it.
-We will give you a zip file containing an access token, a key pair and configuration file.
-The access token is used for communicating with Sandbox API gateway to authenticate you.
+We will authorize you through GitHub OAuth to grant you access to the Sandbox environment.
+Please visit [our sandbox site](https://scalar-labs.com/sandbox/), read the [terms of use](https://scalar-labs.com/terms-of-use), and press the button `Try Now`.
+We will provide you a zip file containing the necessary access token, key pair and configuration file.
```

Comment:
```suggestion
We will provide you with a zip file containing the necessary access token, key pair and configuration file.
```

By the way, is it really a key pair? Or certificate and private key?

Reply:
I agree with "a key pair".

I think that the standard meaning of "key pair" means a public and private key. See, e.g., https://docs.oracle.com/javase/7/docs/api/java/security/KeyPair.html. If here it means a certificate and a private key, I think this should be written explicitly.